src/mayur/BankingSystem.java

```diff
@@ -0,0 +1,66 @@
+package mayur;
+
+public class BankingSystem {
+	
+		 int debitCount = 0;
```

Comment:
Not require to give 0 value to variable, if you are not initialised any variable with any value then it will initialised with it's default value.